snmp/datadog_checks/snmp/data/conf.yaml.example

```
@@ -219,83 +228,111 @@ instances:
-    ## Number of times a discovered host can fail before we remove it from the discovered list.
-    ## Only available using python SNMP integration
+    ## Number of times a discovered host can fail before it is removed from the discovered list.
+    ## Only available using python SNMP integration.
     #
     # discovery_allowed_failures: 3
 
     ## @param workers - integer - optional - default: 5
     ## Number of workers used for check when using discovery.
-    ## Only available using python SNMP integration
+    ## Only available using python SNMP integration.
     #
     # workers: 5
 
     ## @param enforce_mib_constraints - boolean - optional - default: true
-    ## If set to false we will not check the values returned meet the MIB constraints.
-    ## Only available using python SNMP integration
+    ## If set to false, the the values returned are not checked to ensure they meet the MIB constraints.
+    ## Only available using python SNMP integration.
     #
     # enforce_mib_constraints: true
 
     ## @param bulk_threshold - integer - optional - default: 0
     ## The number of symbols in each table that triggers a BULK request.
     ## Only relevant for SNMP > 1.
-    ## Only available using python SNMP integration
+    ## Only available using python SNMP integration.
     #
     # bulk_threshold: 0
 
     ## @param refresh_oids_cache_interval - integer - optional - default: 0
-    ## (beta feature) Set this option to enable caching of OIDs. The value is the number of seconds before the
+    ## Note: Beta feature, only available using python SNMP integration.
+    ## Set this option to enable caching of OIDs. The value is the number of seconds before the
     ## the cache is refreshed.
     ## This option works best when `oid_batch_size` is set to 128 or 256.
     ## This is useful to improve performance when fetching metrics from large tables.
     ## When OID caching is enabled, table row OIDs are reused. This means that the table can be fetched
     ## using batched GET queries, as opposed to one GETNEXT query per row in the table.
     ## The caching is disabled if the interval is set to `0` (default).
     ##
-    ## Caveat: If some tables rows are dynamic, new rows will only be fetched at every refresh interval.
-    ##         - This means new OIDs will only be picked up when OIDs are refreshed according to the refresh interval.
+    ## Note: If some tables rows are dynamic, new rows are only fetched at every refresh interval.
+    ##         - This means new OIDs are only picked up when OIDs are refreshed according to the refresh interval.
     ##         - With auto discovery, if the same IP is assigned to a different device with different OIDs,
-    ##         the OIDs will be refreshed only according to the refresh interval.
-    ## Only available using python SNMP integration
+    ##         the OIDs are refreshed only according to the refresh interval.
+    #
+    # refresh_oids_cache_interval: 0
```

```suggestion
    # refresh_oids_cache_interval: 3600
```

Let's keep the same display value as before